src/utilities/utils.js

```diff
@@ -7,23 +7,50 @@ export const cleanData = data => {
       colors: currentPolish.product_colors
     }
   })
-  console.log(freshData)
 
-//reduce through the array
-//inside reduce, filter through the current array's polish brands
-//return only the brands with duplicates in array
-//then, map over this array of duplicates to create a new array of all colors
-//.flat that bb
-//create a new array without duplicates using Set
-//reset the currentPolishes colors to be this new Set
-//push the currentPolish into the accumulator
-//return the reduce
+  const findUnrepeated = array => [...new Set(array)]
```

Comment:
Cool! I haven't used Set before but it looks interesting (MDN)

Reply:
Super handy! Loving it so far.